jbpm-bpmn2/src/test/resources/BPMN2-SimpleXMLProcess.bpmn2

```diff
@@ -12,7 +12,10 @@
              xmlns:tns="http://www.jboss.org/drools">
 
   <process processType="Private" isExecutable="true" id="Minimal" name="&lt; &amp; &gt;" tns:packageName="com.sample" >
-
+    <extensionElements>
+     <tns:import name="org.kie.api.runtime.process.CaseAssignment" />
+     <tns:import name="org.kie.api.runtime.process.CaseData" />
+    </extensionElements>
```

Comment:
Was this addition needed? I mean this process definition is used only in BPMN2XMLTest.java and I cannot see any reason why this addition is needed. Or does it just test a use case with custom imports?

Reply:
It compares two bpmn by xml and since this pr adds two imports automatically they will show up in dump of the process